### PR TITLE
do not ignore errors in tests

### PR DESCRIPTION
**Pull request by harshavardhana**
- update eslint rules for comments
- fix comment format

### CODE REVIEW

**Review by prakashsvmx — APPROVED** [review]
Ran tests locally and verified
Ran mint tests locally and verified. 
Changes look good to me. :+1:

**Comment by harshavardhana** [comment]
PTAL @kanagarajkm

File: src/main/minio.js [hunk 1]
```diff
@@ -941,14 +945,14 @@ export class Client {
       headers.range = range
     }
 
-    var expectedStatus = 200
+    var expectedStatusCodes = [200]
     if (range) {
-      expectedStatus = 206
+      expectedStatusCodes.push(206)
```
Comment (kannappanr) [inline src/main/minio.js:950]: if range is set, the status code cannot be 200 right

Comment (harshavardhana) [reply src/main/minio.js:950]: It is set such that 200 is allowed. HTTP spec doesn't seem to explicit deny it.

**Review by kanagarajkm — APPROVED** [review]

**Pull request merged**